docs/index.md

```diff
@@ -37,6 +37,16 @@ By default iOS and Android projects are created. For WPF apps use `--WPF`, for e
    Android only: dotnet new fabulous-app -n SqueakyApp --iOS=false
    iOS only:     dotnet new fabulous-app -n SqueakyApp --Android=false
    WPF only:     dotnet new fabulous-app -n SqueakyApp --WPF --Android=false --iOS=false
+   
+5. If you are using Visual Studio for Mac and you want to start with File -> New make sure you target ".NE Standard" to add the references to Fabulous:
```

Comment:
Small typo: ".NE Standard" => ".NET Standard"
Could you also add a comma between `File -> New` and `make sure ...` to make it easier to read?